doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
@chu11:  just changed prototypes for `treeobj_decode_val()` and `flux_kvs_lookup_get_raw()` in the branch we're working on and pushed.